src/vcpkg/export.ifw.cpp

```diff
@@ -424,19 +420,17 @@ namespace vcpkg::Export::IFW
         const auto ifw_packages_dir_path = get_packages_dir_path(export_id, ifw_options, paths);
 
         fs.remove_all(ifw_packages_dir_path, ec, failure_point);
-        Checks::check_exit(VCPKG_LINE_INFO,
-                           !ec,
-                           "Could not remove outdated packages directory %s due to file %s",
-                           ifw_packages_dir_path,
-                           failure_point);
+        Checks::msg_check_exit(
+            VCPKG_LINE_INFO,
+            !ec,
+            msg::format(msgRemovingPackageDirFailed, msg::path = ifw_packages_dir_path).append_raw(failure_point));
 
         fs.create_directory(ifw_packages_dir_path, ec);
-        Checks::check_exit(VCPKG_LINE_INFO, !ec, "Could not create packages directory %s", ifw_packages_dir_path);
+        Checks::msg_check_exit(VCPKG_LINE_INFO, !ec, msgCreatingPackageDirFailed, msg::path = ifw_packages_dir_path);
```

Comment:
Ditto I think this should call `fs.create_directory(ifw_packages_dir_path, VCPKG_LINE_INFO)`.